src/openforms/contrib/digid_eherkenning/utils.py

```diff
@@ -19,3 +26,22 @@ def get_eherkenning_logo(request) -> Dict[str, str]:
         "href": "https://www.eherkenning.nl/",
         "appearance": LogoAppearance.light,
     }
+
+
+def create_digid_eherkenning_csp_settings(
+    config: DigidConfiguration | EherkenningConfiguration, config_type: str
+) -> None:
+    if not config.metadata_file_source:
+        return
+
+    # create the new directives based on the POST bindings of the metadata XML and
+    # the additional constants
+    urls, _ = config.process_metadata_from_xml_source()
+    if ADDITIONAL_CSP_VALUES[config_type]:
+        urls = (
+            f"{ADDITIONAL_CSP_VALUES[config_type]} {urls['sso_url']} {urls['slo_url']}"
+        )
+    else:
+        urls = f"{urls['sso_url']} {urls['slo_url']}"
```

Comment:
```suggestion
    csp_values = [urls["sso_url"], urls["slo_url"]]
    if (additional_csp_values := ADDITIONAL_CSP_VALUES.get(type(config), [])):
        csp_values += additional_csp_values
   form_action_urls = " ".join(csp_values)
   ...
```
you can reduce the branching and avoid the duplication with the f-string arguments - just build a list of relevant URLs and at the end join it with spaces & then pass that to the `set_for` method.

Additionally, you start out with the `urls` variable from the digid-herkenning library which is a dictionary `dict[str, str]`, but then this variable becomes a `str`. While this is valid python, my type checker will definitely complain about it :) It's better to use a different variable and stick to "static" types at runtime.